src/reynir_correct/checker.py

```diff
@@ -395,29 +401,31 @@ def create_sentence(self, job: Job, s: TokenList) -> Sentence:
         return sent
 
 
-def check_single(sentence_text: str, **options: Any) -> Optional[Sentence]:
+def check_single(
+    sentence_text: str, rc: GreynirCorrect, **options: Any
```

Comment:
I'm a bit wary about this non-compatible API change. Consider using

`def check_single(sentence_text: str, *, rc: Optional[GreynirCorrect] = None, **options: Any) -> Optional[Sentence]:`

This forces rc to be a keyword parameter, with a default of None. If rc is None, we would revert to the previous behavior (global rc instance?).

Reply:
Fixed, now rc is optional